tutorials/navigation/navigation_advanced.rst

```diff
@@ -0,0 +1,12 @@
+.. _doc_navigation_advanced:
```

Comment:
Did you mean to add this page now?

Reply:
No I didn't, this is a leftover from an old iteration of the complete docs.